sklearn/ensemble/gradient_boosting.py

```diff
@@ -44,7 +44,7 @@
 from time import time
 from ..model_selection import train_test_split
 from ..tree.tree import DecisionTreeRegressor
-from ..tree._tree import DTYPE
+from ..tree._tree import DTYPE, DOUBLE
```

Comment:
It is interesting how this is our naming convention for the dtypes of X and y. In the future we may consider `X_DTYPE` and `Y_DTYPE`.

Reply:
Yeah I know. You're the second +1 here now @thomasjpfan, wanna push the green button? ;)